vars/burgrNotify.groovy

```diff
@@ -1,25 +1,32 @@
 #!/usr/bin/groovy
 
-def call(url, branch, commit, step, type, status, timestamp) {
-  echo "Send a step notification to BURGR: [repo: ${url}, branch: ${branch}, commit: ${commit}, step: ${step}, type: ${type}, status: ${status}]"
-  def data = burgrExtractDataFromURL(url, timestamp)
-  def message = """
+def call(owner, project, pipeline, branch, commit, step, type, status, started_at, finished_at) {
```

Comment:
pipeline seems to be too BURGR oriented.
I would call it buildNumber to ease the understanding of this field for the caller

Reply:
Done